Automattic/wp-calypso
# Marketplace: Rename Marketplace Installation component to work with multiple products

Fixes #75625

## Proposed Changes

* Rename the Marketplace Product Installation folder
* Update variable names to match plugins or themes instead of a general name
* Update the page title from `"Plugin installation"` to `"Marketplace installation"`

## Testing Instructions
* Install a plugin or a theme and verify it still working as in production
* The only visible change should be the page title update from `"Plugin installation"` to `"Marketplace installation"`

| Page  | Text change |
| ------------- | ------------- |
|<img width="1271" alt="CleanShot 2023-04-12 at 09 37 53@2x" src="https://user-images.githubusercontent.com/5039531/231459771-e63adaa6-ab4d-4b73-a1d9-f486cd7d88ea.png">|![CleanShot 2023-04-12 at 09 38 24@2x](https://user-images.githubusercontent.com/5039531/231459804-310d7a55-a2f2-4e60-99f8-abbfa3446cae.png)|


## Pre-merge Checklist
